src/plugins/github/edges.js

```diff
@@ -47,12 +50,17 @@ export type ReferencesAddress = {|
   +referrer: GithubNode.TextContentAddress,
   +referent: GithubNode.ReferentAddress,
 |};
+export type MentionsAuthorAddress = {|
+  +type: typeof MENTIONS_AUTHOR_TYPE,
+  +reference: MentionsAuthorReference,
```

Comment:
Interesting decision to tie this to the reference type, as opposed to
transcluding the fields directly. If the property names on the reference
type are changed/added/removed, then Flow will catch an error on line
216 of `edges.js`, right? If so, this is fine with me.

Reply:
Correct.